### DISCUSSION TITLE
Comments and archive records

### DISCUSSION THREAD

**Discussion by Giorgos-Hellene** [YetiForceCompany/YetiForceCRM#15392] [General] [answered]
Hi company !! i hope you are fine !

i have a question about Comments and Archived record.

My test is here https://gitdeveloper.yetiforce.com

After writing a comment on any module there are four options about it. 
1) Reply
2) edit
3) archive record
4) Move to Recycle bin
The situations 1,2 i understand it.

i need your help about 3 and 4 and one more.

1) When a comment is marked as Archive record what is going on ? Where is this comment if i click archive record and after ok?
Also if i move the comment to Recycle bin where can i find it ?

2) I try to find the movement in updates, on Summary module but i can't. Sorry if this issue i must write it on Issue . Tell me if it is neseccary. 

3) At last tell me please how works Current Record and Related Records that there is above.

Thank you !!

**Comment by rskrzypczak** [comment, marked as answer]
> When a comment is marked as Archive record what is going on ? 

  Comment is marked as Archive and is not displayed anywhere.
> Also if i move the comment to Recycle bin where can i find it ?

  Unfortunately, there is no access to it. The Comments module does not have a list view and the trash can view does not support this module.

> I try to find the movement in updates, on Summary module but i can't.

  Yes. Currently, you will not see any base record activity related to comments other than adding a new comment.
  However, you can see the activity in general in the comments on the Updates widget (Dashboards)
![image](https://user-images.githubusercontent.com/10221999/141816509-7115de96-b591-4666-92a1-ad149ae4cfe0.png)

> At last tell me please how works Current Record and Related Records that there is above.

 Current Record - these are comments created directly on this record
 Related Records - these are comments created in related records (it only includes modules lower in the module hierarchy - a list of specific modules)

**Reply by Giorgos-Hellene** [reply]
Thank you !!
